courses/management/commands/refund_enrollment.py

```diff
@@ -33,6 +38,7 @@ def handle(self, *args, **options):
         """Handle command execution"""
         user = fetch_user(options["user"])
```

Comment:
I'm getting an exception here if I leave out `user`. Does it make sense for make `user` a required argument?